arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentTableViewController.swift

```diff
@@ -205,25 +195,19 @@ class ContentTableViewController: UITableViewController {
 
     }
     
-    @objc
-    func expandCell(_ sender: UIButton) {
-        updateExpandedRow(IndexPath(row: sender.tag, section: 0), collapseIfSelected: true)
-    }
-    
     private func updateExpandedRow(_ indexPath: IndexPath, collapseIfSelected: Bool) {
```

Comment:
A better name for this function might be `toggleExpansion(at:)`. It looks like `true` is always passed for `collapseIfSelected`, so that parameter probably isn't needed.

Reply:
Good point!